de.fu_berlin.inf.dpp.stf.test/sonar-project.properties

```diff
@@ -0,0 +1,16 @@
+sonar.projectName        = Saros/Eclipse
```

Comment:
You have to use a name that is unique in the project `Saros/Eclipse` is already used.